File: src/test/java/com/raychenon/leetcode/IntegerToRomanTest.java
package com.raychenon.leetcode;

import org.junit.BeforeClass;
import org.junit.Test;

import static junit.framework.TestCase.assertEquals;

/**
 * User: raychenon
 * Date: 2019-09-12
 * https://leetcode.com/problems/integer-to-roman/
 */
public class IntegerToRomanTest {

    private static IntegerToRoman integerToRoman;

    @BeforeClass
    public static void setUp() {
        integerToRoman = new IntegerToRoman();
    }

    @Test
    public void test9() {
        assertEquals("IX", integerToRoman.intToRoman(9));
    }

    @Test
    public void test40() {
        assertEquals("XL", integerToRoman.intToRoman(40));
    }

    @Test
    public void test58() {
        assertEquals("LVIII", integerToRoman.intToRoman(58));
    }

    @Test
    public void test1994() {
        assertEquals("MCMXCIV", integerToRoman.intToRoman(1994));
    }

    @Test
    public void test3999() {
        assertEquals("MMMCMXCIX", integerToRoman.intToRoman(3999));
    }

}

File: src/test/kotlin/com/raychenon/kotlin/leetcode/IntegerToRomanTest.kt
package com.raychenon.kotlin.leetcode

import org.junit.Assert
import org.junit.Test

/**
 * User: raychenon
 * Date: 2019-09-12
 * https://leetcode.com/problems/integer-to-roman
 */
class IntegerToRomanTest {

    @Test
    fun test1() {
        Assert.assertEquals("I", IntegerToRoman.intToRoman(1))
    }

    @Test
    fun test9() {
        Assert.assertEquals("IX", IntegerToRoman.intToRoman(9))
    }

    @Test
    fun test48() {
        Assert.assertEquals("XLVIII", IntegerToRoman.intToRoman(48))
    }

    @Test
    fun test58() {
        Assert.assertEquals("LVIII", IntegerToRoman.intToRoman(58))
    }

    @Test
    fun test1994() {
        Assert.assertEquals("MCMXCIV", IntegerToRoman.intToRoman(1994))
    }

    @Test
    fun test3999() {
        Assert.assertEquals("MMMCMXCIX", IntegerToRoman.intToRoman(3999))
    }
}